src/main/java/uk/gov/hmcts/ccd/data/caseaccess/CaseRoleService.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.ccd.data.caseaccess;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.data.user.CachedUserRepository;
+import uk.gov.hmcts.ccd.data.user.UserRepository;
+
+@Service
+public class CaseRoleService {
```

Comment:
Couldn't this be merged with the already existing `CaseAccessService`? (https://github.com/hmcts/ccd-data-store-api/blob/master/src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseAccessService.java)

It's got the same dependencies, and is responsible for case access, which is a superset of case roles.

Reply:
done